packages/picker/src/Picker.ts

```diff
@@ -105,6 +107,9 @@ export class PickerBase extends SizedMixin(Focusable) {
     @property({ type: Boolean, reflect: true })
     public invalid = false;
 
+    @property({ type: Boolean, reflect: true })
+    public loading = false;
```

Comment:
I think we should normalize this to the API recently added to `<sp-button>`, [pending](https://fix-switch-onchange-event--spectrum-web-components.netlify.app/components/button/#pending).